syntax = "proto2";

package keras_classification.protos;

message Model {
    enum Type {
        RESNET50 = 1;
    }

    // height and width of input images
    required int32 input_size = 1;
    required int32 nb_classes = 2;
    required Type type = 3;
    required string model_path = 4;
}
